Tema1/Animals/Animals/IAnimal.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Animals
+{
+    public interface IAnimal
+    {
+        void makeSound();

```

Comment:
din cate stiu eu naming ul in C# spune ca o metoda este cu upper case MakeSound()